sheet_to_triples/run.py

```diff
@@ -100,7 +103,8 @@ def _get_books(self, book):
 
     def _iter_data(self, tf):
         if self.books and tf.uses_sheet():
-            row_iter = xl.iter_sheet(self._get_books(tf.book), tf.sheet)
+            row_iter = xl.iter_sheet(
+                self._get_books(tf.book), tf.sheet, self.sheet_encoding)
```

Comment:
I think a better interface, rather than taking the encoding from the commandline and passing in through run, would be have `sheet_encoding` as an (optional) attribute on the transform, then this passes it through as `tf.sheet_encoding` instead of `self.sheet_encoding` here.